src/docs/event.yaml

```diff
@@ -603,6 +603,40 @@ paths:
                     example: User already joined
         500:
           description: Internal server error
+  /api/event/{id}/invite-volunteer:
+    post:
+      tags:
+        - Event
+      summary: Invite Volunteer
+      description: Invite Volunteer that share the same interests
```

Comment:
This endpoint requires authentication, to indicate this (by adding a lock icon next to the endpoint), we add the following 2 lines: 
```
      security:
        - auth_cookie: []
```